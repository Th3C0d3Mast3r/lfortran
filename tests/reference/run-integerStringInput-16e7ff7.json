{
    "basename": "run-integerStringInput-16e7ff7",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/integerStringInput.f90",
    "infile_hash": "d1381b4a9fe635ed7e76dec66144da99b0aa49ffa9588db854a562aa",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-integerStringInput-16e7ff7.stderr",
    "stderr_hash": "93103ec86aa429ca52c869eca8168e2a245d3dec573aec6aaf204e65",
    "returncode": 1
}