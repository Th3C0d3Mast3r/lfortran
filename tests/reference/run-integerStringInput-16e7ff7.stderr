warning: Argument `action` isn't supported yet
 --> tests/errors/integerStringInput.f90:4:5
  |
4 |     open(unit=10, file="invalidInput_integer.txt", status="old", action="read")
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ignored for now
